its/ruling/src/integrationTest/java/org/sonarsource/iac/IacRulingTest.java

```diff
@@ -109,6 +109,14 @@ void testKubernetes() throws IOException {
     runRulingTest("kubernetes", properties);
   }
 
+  @Test
+  void testKubernetesCrossFile() throws IOException {
```

Comment:
Why separate test instead of including to existing Kubernetes test?

Reply:
Because the LimitRange would count for all K8s files. That way we would not see any limit issue anymore. Not ideal, but we have to set in place some project-context testing.